### PR TITLE
To extract all user's DDL & privs for MT database at pre-refresh and recreating them back at post-refresh

### PR DESCRIPTION
**Pull request by soodabhinav**
# Problem:

# Solution:

# Testing

### CODE REVIEW

File: DBTEMPLATE/post-refresh/scripts/dxh_wrapper_script.sh [hunk 1]
```diff
@@ -47,6 +47,8 @@ HOOK_OP=${4}
 # sample code to optimize a VDB for Masking (expects ORACLE_SID and ORACLE_HOME to be set)
 # ${BASEDIR}/ora_vdb_masking_prep.sh 
 
+#. $BASEDIR/recreate_users_MT.sh $DELPHIX_PDB_NAME
```
Comment (Ranzo3) [inline DBTEMPLATE/post-refresh/scripts/dxh_wrapper_script.sh:50]: I don't think we want a  "." here.  If it's an environment file, we use "." or `source`.  

Technically, I think your command will work fine, it's just confusing style (to me at least).  Unless we are sure the "." is needed (in other words, you want to run this from the current shell) we should take it out.

https://superuser.com/questions/176783/what-is-the-difference-between-executing-a-bash-script-vs-sourcing-it#:~:text=7%20Answers&text=Sourcing%20a%20script%20will%20run,in%20your%20currently%20running%20shell.

File: DBTEMPLATE/pre-refresh/scripts/dxh_wrapper_script.sh [hunk 2]
```diff
@@ -40,6 +40,7 @@ HOOK_OP=${4}
 
 # sample code to initiate impdp
 #/softwares/scripts/hooks/dxh_impdp.sh -o ${HOOK_OP} -d '/softwares/scripts/hooks/dumpdir' -p DBTEMPLATE.impdp.delphix.par 
+#. $BASEDIR/extract_users_ddl_privs.sh $DELPHIX_PDB_NAME
```
Comment (Ranzo3) [inline DBTEMPLATE/pre-refresh/scripts/dxh_wrapper_script.sh:43]: Same as above.  Just remove the "."

**Review by Ranzo3 — CHANGES_REQUESTED** [review]
I don't think we want a "." here, but otherwise looks great!

File: DBTEMPLATE/pre-refresh/scripts/dxh_wrapper_script.sh (hunk 2 above)
Comment (soodabhinav) [reply DBTEMPLATE/pre-refresh/scripts/dxh_wrapper_script.sh]: changes has been done as suggested

**Pull request merged**